### ISSUE TITLE
Could not decorate Pull Request on Github

### ISSUE THREAD

**Issue by ndeitch** [mc1arke/sonarqube-community-branch-plugin#90]
**Describe the bug**
Github decorator not working

**To Reproduce**
1. Run scanner:
```bash
sonar-scanner \
-Dsonar.projectKey=sample-ms \
-Dsonar.host.url=http... \
-Dsonar.login=... \
-Dsonar.pullrequest.key=3 \
-Dsonar.pullrequest.branch=feature/new_endpoint \
-Dsonar.pullrequest.base=master
```
2. Check sonarqube `ce.log` file:
```
2020.02.17 09:14:04 INFO  ce[AXBSbQ3vn21drEDcQ0_h][c.g.m.s.p.c.p.PullRequestPostAnalysisTask] using pull request decoratorGithub
2020.02.17 09:14:05 ERROR ce[AXBSbQ3vn21drEDcQ0_h][o.s.c.t.p.a.p.PostProjectAnalysisTasksExecutor] Execution of task class com.github.mc1arke.sonarqube.plugin.ce.pullrequest.PullRequestPostAnalysisTask failed
java.lang.IllegalStateException: Could not decorate Pull Request on Github
	at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.github.GithubPullRequestDecorator.decorateQualityGateStatus(GithubPullRequestDecorator.java:41)
	at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.PullRequestPostAnalysisTask.finished(PullRequestPostAnalysisTask.java:161)
	at org.sonar.ce.task.projectanalysis.api.posttask.PostProjectAnalysisTasksExecutor.executeTask(PostProjectAnalysisTasksExecutor.java:118)
	at org.sonar.ce.task.projectanalysis.api.posttask.PostProjectAnalysisTasksExecutor.finished(PostProjectAnalysisTasksExecutor.java:109)
	at org.sonar.ce.task.step.ComputationStepExecutor.executeListener(ComputationStepExecutor.java:91)
	at org.sonar.ce.task.step.ComputationStepExecutor.execute(ComputationStepExecutor.java:63)
	at org.sonar.ce.task.projectanalysis.taskprocessor.ReportTaskProcessor.process(ReportTaskProcessor.java:81)
	at org.sonar.ce.taskprocessor.CeWorkerImpl$ExecuteTask.executeTask(CeWorkerImpl.java:209)
	at org.sonar.ce.taskprocessor.CeWorkerImpl$ExecuteTask.run(CeWorkerImpl.java:191)
	at org.sonar.ce.taskprocessor.CeWorkerImpl.findAndProcessTask(CeWorkerImpl.java:158)
	at org.sonar.ce.taskprocessor.CeWorkerImpl$TrackRunningState.get(CeWorkerImpl.java:133)
	at org.sonar.ce.taskprocessor.CeWorkerImpl.call(CeWorkerImpl.java:85)
	at org.sonar.ce.taskprocessor.CeWorkerImpl.call(CeWorkerImpl.java:53)
	at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
	at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:69)
	at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.NullPointerException: null
	at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.github.v3.RestApplicationAuthenticationProvider.getInstallationToken(RestApplicationAuthenticationProvider.java:114)
	at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.github.v4.GraphqlCheckRunProvider.createCheckRun(GraphqlCheckRunProvider.java:96)
	at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.github.GithubPullRequestDecorator.decorateQualityGateStatus(GithubPullRequestDecorator.java:39)
	... 21 common frames omitted
2020.02.17 09:14:05 INFO  ce[AXBSbQ3vn21drEDcQ0_h][o.s.c.t.p.a.p.PostProjectAnalysisTasksExecutor] Pull Request Decoration | status=FAILED | time=1326ms
2020.02.17 09:14:05 INFO  ce[AXBSbQ3vn21drEDcQ0_h][o.s.c.t.CeWorkerImpl] Executed task | project=sample-ms | type=REPORT | pullRequest=3 | id=AXBSbQ3vn21drEDcQ0_h | submitter=admin | status=SUCCESS | time=1856ms
```
**Expected behavior**
Pull request decorator on github pull request.

**Software Versions**
 - SonarQube Version: Version 8.1 (build 31237)
 - Plugin Version: custom build from master at commit fdb382a064b4526f7c34cbcda658306e7c2c723c

**Additional context**
https://github.com/mc1arke/sonarqube-community-branch-plugin/blob/fdb382a064b4526f7c34cbcda658306e7c2c723c/src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/github/v3/RestApplicationAuthenticationProvider.java#L114 `projectPath` is null, I tried to debug why with no success.

I guess there is some configuration missing, but I don't know which one.

**Comment by mc1arke** [comment]
I think this may be an error in the code: 
https://github.com/mc1arke/sonarqube-community-branch-plugin/blob/fdb382a064b4526f7c34cbcda658306e7c2c723c/src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/github/v4/GraphqlCheckRunProvider.java#L92 reads the ALM Slug as the project path, which is set in the Github Binding WS: https://github.com/mc1arke/sonarqube-community-branch-plugin/blob/fdb382a064b4526f7c34cbcda658306e7c2c723c/src/main/java/com/github/mc1arke/sonarqube/plugin/server/pullrequest/ws/action/github/SetGithubBindingAction.java#L67
I think that that first block should be `String projectPath = projectAlmSettings.getAlmRepo()`, although I can't remember if that path is `project/repo` or just `repo`.

**Comment by ndeitch** [comment]
@mc1arke I made the change you said and it's working. Should I open a PR?

**Comment by mc1arke** [comment]
Yes please.

**Comment by mc1arke** [comment]
Fixed in #92
